### PR TITLE
Updating Supercheq docs

### PR DESCRIPTION
**Pull request by pgoiporia**

### CODE REVIEW

**Comment by stephanielee9** [comment]
A couple things we should probably update:
* Change all instances of Super.tech to Infleqtion
* The Binder and Colab links are currently broken - [this](https://github.com/SupertechLabs/client-superstaq/issues/525) should fix it: `[alt text](image_url)](link_url)`
* Make the install instructions more similar to the [QSCOUT](https://superstaq.readthedocs.io/en/latest/optimizations/qscout/qscout_qss.html) and IBM tutorials
* Somewhat relatedly, I think we should delete `# provider = qss.SuperstaQProvider(api_key="Enter Your API Key Here")` and equivalent text in the `css` notebook to mirror the QSCOUT and IBM install instructions

File: docs/source/supercheq/Supercheq_css.ipynb
Comment (stephanielee9) [inline docs/source/supercheq/Supercheq_css.ipynb:5]: <p>Is it ok to put this button and the next in the same cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (stephanielee9) [inline docs/source/supercheq/Supercheq_css.ipynb:5]: <p>Can you put these all at the <code>h2</code> level? (i.e., <code>##</code> instead of <code>#</code>) This way the headings won't show up in the sidebar.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: docs/source/supercheq/Supercheq_qss.ipynb
Comment (stephanielee9) [inline docs/source/supercheq/Supercheq_qss.ipynb]: <p>Put the buttons in the same line if possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (stephanielee9) [inline docs/source/supercheq/Supercheq_qss.ipynb]: <p>Update to <code>##</code> instead of <code>#</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (stephanielee9) [inline docs/source/supercheq/Supercheq_qss.ipynb]: <p>Update to <code>##</code> instead of <code>#</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by stephanielee9** [comment]
Some minor formatting nits but otherwise, looks good!

File: docs/source/supercheq/Supercheq_qss.ipynb
Comment (pgoiporia) [reply docs/source/supercheq/Supercheq_qss.ipynb]: <p>I'm not sure how to get them in the same line. Even if I put the code in the same markdown cell, the buttons appear below one another.</p>

Comment (stephanielee9) [reply docs/source/supercheq/Supercheq_qss.ipynb]: Strange... They look right in https://github.com/Infleqtion/client-superstaq/pull/551 so maybe just wait until those changes are merged into main? Or copy and paste what Pranav had and maybe that'll fix it

**Review by stephanielee9 — APPROVED** [review]
lgtm - thanks!!

**Pull request merged**
